src/Containers/Loading/Loading.tsx

```diff
@@ -15,12 +15,15 @@ export interface LoadingProps
         MainInterface {
     loading?: boolean;
     message?: string;
+    /** if true, the component is not positioned absolute */
+    isNavigationPageLoader?: boolean;
 }
 
 export const Loading: React.FC<LoadingProps> = ({
     children,
     loading = false,
     message = 'Loading...',
+    isNavigationPageLoader = false,
```

Comment:
Use this prop on the root not individual components remove the fragment.

Reply:
I don't want to break behavior

Reply:
I don't want break any behavior

Reply:
Try it and see maybe you don't break any behaviour we can test it

Reply:
I have not make the Loading component. I don't fully understand it